radio/src/myeeprom.h

```diff
@@ -414,7 +414,13 @@ PACK(typedef struct {
   int16_t failsafeChannels[NUM_CHNOUT];
   int8_t  ppmDelay;
   int8_t  ppmFrameLength;
-  uint8_t ppmPulsePol;
+  uint8_t ppmPulsePol:1;
+#if defined(PCBSKY9X) && defined(REVX)
+  uint8_t ppmOpenDrain:1;
+  uint8_t spare:6;
+#else
+  uint8_t spare:7;
+#endif
```

Comment:
I would even have saved one byte in this structure. But well, it avoids a new shift as you did, let's keep it like that!